tests/ballerina-test-integration/src/test/java/org/ballerinalang/test/util/websocket/client/WebSocketTestClient.java

```diff
@@ -175,6 +174,14 @@ public boolean isPong() {
         return webSocketHandler.isPong();
     }
 
+    /**
+     * Gets the header value from the response headers.
```

Comment:
Shall we rephrase this with the objective? Here headers are being retrieved from webSocketHandler. Same method description is reoccurring in webSocketHandler class